src/js/components/header/header.js

```diff
@@ -299,8 +302,22 @@ const actionCreators = {
 
 const mapStateToProps = state => {
   const plan = state.users.organization ? state.users.organization.plan : 'os';
+  const currentUser = state.users.byId[state.users.currentUser];
+  let allowUserManagement = false;
+  if (currentUser?.roles) {
+    // TODO: move these + additional role checks into selectors
+    const isAdmin = currentUser.roles.some(role => role === 'RBAC_ROLE_PERMIT_ALL');
+    allowUserManagement =
+      isAdmin ||
+      currentUser.roles.some(role =>
+        state.users.rolesById[role]?.permissions.some(
+          permission => permission.action === 'http' && permission.object.value === '/api/management/v1/useradm/.*' && ['any'].includes(permission.object.type)
```

Comment:
Was this discussed with @merlin-northern? I'm not aware of any built-in role with this permission, shall we define a specific role to manage users? From my understanding, the admins are assigned to the role `RBAC_ROLE_PERMIT_ALL`, and this is the only check needed.

Reply:
I am sure we should define a specific role to manage users. we started with what we had, now I think we can move forward.

Reply:
yep, that permission is only given to allow user management, while not giving full admin access...
if it's possible to have the role defined in the backend it would be all the better and I'll happily change it here as well

Reply:
@mzedel ack, as soon as we get good default roles we'll change the UI to take advantage of them.